pandas/core/frame.py

```diff
@@ -3250,48 +3250,34 @@ def assign(self, **kwargs):
 
         Examples
         --------
-        >>> df = pd.DataFrame({'A': range(1, 11), 'B': np.random.randn(10)})
+        >>> df = pd.DataFrame({'temp_c': (17.0, 25.0)},
```

Comment:
Can you use a list instead of a tuple for the data? I think it's more conventional.